content/en/01-about-pixie/03-pixie-ebpf.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Pixie + eBPF"
+metaTitle: "About Pixie | Pixie + eBPF"
+metaDescription: "How Pixie uses eBPF to automatically collect data"
+order: 3
+---
+
+Pixie uses [eBPF](https://www.brendangregg.com/ebpf.html) to drive much its data collection.
+This eBPF-based approach enables Pixie to collect data in a way that requires no user instrumentation
+(i.e. no code changes, no redeployments), and with very low overhead.
+
+Pixie sets up eBPF probes to trigger on a number of kernel or user-space events,
+at which time it collects the data of interest, such as various protocol messages. Additional details on how
+Pixie uses eBPF in its different features are described below.
+
+## Protocol Tracing
+
+One of the main features of Pixie is its ability to automatically trace protocol messages.
+
+When Pixie is deployed to the nodes in your cluster, it deploys eBPF kernel probes (kprobes) that are set-up to trigger
+on Linux syscalls used for networking. Then, when your application makes network-related syscalls  -- such as `send()` and `recv()` --
+Pixie's eBPF probes snoop the data and sends it to the PEM. The data is then parsed according the detected protocol and stored for querying.
+
+::: div image-l
+<svg title='Pixie Protocol Tracing with eBPF' src='pixie-ebpf/protocol_tracing.svg' />
+:::
+
+
+### Tracing TLS/SSL Connections
+
+Pixie's SSL tracing works in a similar way as its basic protocol tracing, with one main difference:
+Instead of using eBPF to snoop the data at `send()` and `recv()`, eBPF user-space probes (uprobes)
+are set-up directly on the TLS library's API. This enables Pixie to capture the data before it is encrypted.
```

Comment:
set up

Reply:
maybe dumb question: also before it is encrypted, isn't it after it is decrypted?

Reply:
We want to capture the data before it it's encrypted so we can see the content.